docs/en/observability/ingest-splunk.asciidoc

```diff
@@ -0,0 +1,131 @@
+REST API is listening on.
+
+The Splunk username and password must be of a user with a role or
+capability to use REST API endpoints.  Administrative users have these
+permissions by default.
+
+SSL Configuration is available under the "Advanced options".  These may be necessary
+if Splunk Enterprise server uses self signed certificates.  See
+https://www.elastic.co/guide/en/beats/filebeat/current/configuration-ssl.html[SSL Options]
+for valid configuration options.
+
+[role="screenshot"]
+image::images/kibana-fleet-third-party-rest-settings.png[{fleet} showing enabling third-party REST API settings]
+
+== Step 4: Enter information to select data from Splunk
+
+For each type of log file enter the interval and Splunk search string.
+
+The interval is expressed as a
+https://golang.org/pkg/time/#ParseDuration[Go duration].  The interval
+is the time between requests sent to the Splunk Enterprise REST API to
+request new information.  Intervals less than one second are not
+recommended, Splunk only maintains second accuracy for index time.
+The interval should closely match the rate at which data arrives at
+the Splunk Enterprise Server.  For example an interval of "5s" for
+data that only arrives at the Splunk Enterprise Server every hour will
+generate unnecessary load on the Splunk Enterprise Server.
+
+The search string is the Splunk search used to uniquely describe the
+events that match the type of log file you are trying to configure.
+For example, to uniquely describe Nginx access logs "search
+sourcetype=nginx:plus:access" might be used.  Note, the search string
+must begin with "search" for details refer to the Splunk REST API
+manual and the "search/jobs/export" endpoint.
+
+Be aware that each time the {agent} connects to the Splunk Enterprise
+REST API a Splunk search is performed.  Because of this you want to be
+sure your search string is as specific as possible, since this reduces
+the load on the Splunk Enterprise Server.
+
+Tags may be added in the "Advanced options".  For example, if you'd
+like to tag events coming from Splunk with a 'Splunk' tag, you can add
+it here.  By default, the forward tag is present to indicate that
+events are being forwarded via an intermediary, i.e. Splunk.
+
+[role="screenshot"]
+image::images/kibana-fleet-third-party-rest-dataset-settings.png[{fleet} showing enabling third-party REST API settings]
+
+== Step 5: Save Integration
+
+Click Save Integration
+
+Data and Dashboards will be available just as if you had collected
+the data on the nginx host using log files.
+
+
+[discrete]
+=== Considerations / Questions
+
+The time on the host running the agent and the Splunk Enterprise
+Server should be synchronized to the same time source, with correct
+timezone information.  Failure to do this could result in delays in
+transferring data or gaps in the data received.
+
+Does the Splunk data need to be in a specific format or mapped to
+Splunk's Common Information Model?  No, because these integrations
```

Comment:
It might make sense to bold the questions so they stand out?

```suggestion
**Does the Splunk data need to be in a specific format or mapped to
Splunk's Common Information Model?**  No, because these integrations
```